ui/imports/Utils.qml

```diff
@@ -289,32 +289,85 @@ QtObject {
         return (/( |\t|\n|\r)/.test(c))
     }
 
-    function getLinkDataForStatusLinks(link) {
-        if (!link.includes(Constants.deepLinkPrefix) && !link.includes(Constants.joinStatusLink)) {
-            return
+    function getLinkTitleAndCb(link) {
+        const result = {
+            title: "Status",
```

Comment:
qsTr? dunno if Status is translated

Reply:
I don't think we should translate "Status" since it's the name of the company and app